sklearn/utils/tests/test_sparsefuncs.py

```diff
@@ -94,7 +94,9 @@ def test_incr_mean_variance_axis():
         rng = np.random.RandomState(0)
         n_features = 50
         n_samples = 10
-        data_chunks = [rng.random_integers(0, 1, size=n_features)
+        #data_chunks = [rng.random_integers(0, 1, size=n_features)
+        #               for i in range(n_samples)]
+        data_chunks = [rng.rand(n_features)
```

Comment:
I modify this test in order to pass the test since I find that once I modify these functions to support fused types, they no longer support input with `int` type.

Original implementations can handle `int` type data since they convert input into `np.float64` no matter what original input dtype is.
However, now I have specified `floating` as input datatype of these functions and these functions cannot handle `int` input anymore.

Would reviewers please give me some suggestions?